glue/bin/run-miral

```diff
@@ -14,4 +14,4 @@ export MIR_SERVER_PLATFORM_PATH=${SNAP}/usr/lib/${ARCH}/mir/server-platform
 
 mkdir -p $XDG_RUNTIME_DIR # make sure it exists
 
-exec ${SNAP}/usr/bin/miral-kiosk --console-provider=vt --vt 4 --arw-file --file /run/mir_socket
+exec ${SNAP}/usr/bin/miral-kiosk $*
```

Comment:
Wouldn't `"$@"` be more appropriate here? According to `bash manual`,

> You almost always want a quoted "$@"

!

Reply:
Fixed